planner/core/integration_test.go

```diff
@@ -5332,3 +5332,30 @@ func TestIssue45033(t *testing.T) {
 				                        from   t3 alias3) alias4
 				                where  alias4.c2 = alias2.alias_col1);`).Check(testkit.Rows("0"))
 }
+
+// https://github.com/pingcap/tidb/issues/45036
+func TestIssue45036(t *testing.T) {
```

Comment:
If this test cannot be categorized specifically, it can be placed under ```planner/core/issuetest```.

Reply:
Done